docs/attributes-registry/README.md

```diff
@@ -76,6 +76,7 @@ Currently, the following namespaces exist:
 - [Peer](peer.md)
 - [Process](process.md)
 - [RPC](rpc.md)
+- [Rule](rule.md)
```

Comment:
```suggestion
- [Security Rule](rule.md)
```

To not create confusion/wrong expectations? Also the file name (rule.md)?

Reply:
It has been updated automatically now through the namespace changes